src/tests/tests/asan-error-parse.rkt

```diff
@@ -0,0 +1,248 @@
+#lang racket
+
+(require rackunit
+         racket/serialize
+         seashell/backend/asan-error-parse
+         seashell/backend/project
+         seashell/backend/runner
+         json)
+
+;; Convenience function for creating a project, compiling,
+;; running, and waiting for it to finish. Returns the ASAN
+;; output as a JSON.
+(define (compile-run-wait code [project-name (symbol->string (gensym 'project))])
+  (new-project project-name)
```

Comment:
Use this to use a local copy of the default template:
`(new-project-from project-name (format "file://~a/src/tests/template.zip" SEASHELL_SOURCE_PATH))`